wasm/src/main/java/com/dylibso/chicory/wasm/types/NameCustomSection.java

```diff
@@ -38,15 +38,15 @@ public NameCustomSection(
             List<NameEntry> dataNames,
             List<NameEntry> tagNames) {
         this.moduleName = moduleName;
```

Comment:
Check for null

Reply:
done for the rest of the fields, `moduleName` can be `null` and I converted it to `Optional`